prodtests/full-system-test/dpl-workflow.sh

```diff
@@ -352,7 +352,12 @@ if has_processing_step MUON_SYNC_RECO; then
     elif [[ $RUNTYPE == "PHYSICS" && $BEAMTYPE == "pp" ]] || [[ $RUNTYPE == "COSMICS" ]]; then
       CONFIG_EXTRA_PROCESS_o2_mch_reco_workflow="MCHTracking.chamberResolutionX=0.4;MCHTracking.chamberResolutionY=0.4;MCHTracking.sigmaCutForTracking=7.;MCHTracking.sigmaCutForImprovement=6.;"
     fi
-    has_detector_reco ITS && [[ $RUNTYPE != "COSMICS" ]] && CONFIG_EXTRA_PROCESS_o2_mch_reco_workflow+="MCHTimeClusterizer.irFramesOnly=true;"
+    if has_detector_reco ITS && [[ $RUNTYPE != "COSMICS" ]]; then
+        CONFIG_EXTRA_PROCESS_o2_mch_reco_workflow+="MCHTimeClusterizer.irFramesOnly=true;"
+        [[ -z ${CUT_RANDOM_FRACTION_MCH:-} ]] && CUT_RANDOM_FRACTION_MCH=${CUT_RANDOM_FRACTION_MCH_WITH_ITS:-}
```

Comment:
this seems wrong, I assume you want to check if the _ITS variable is not empty?

Reply:
Actually not... I want to set `CUT_RANDOM_FRACTION_MCH` from `CUT_RANDOM_FRACTION_MCH_WITH_ITS` only if `CUT_RANDOM_FRACTION_MCH` is not set already.

In other words, I want to be still able to set something like `CUT_RANDOM_FRACTION_MCH=0.5` in the extra ENV variables to completely override the defaults.

Does it make sense?

Reply:
ok, fine with me.
But please check in both assignments that CUT_RANDOM_FRACTION_MCH_WITH_ITS or CUT_RANDOM_FRACTION_MCH_NO_ITS respectively are set. So we don't access an undefined variable accidentally.

Reply:
@davidrohr sorry, I missed your comment... doesn't the `:-` construct in the variables access guarantee that a default empty value is used if the variable is not defined?